### PR TITLE
Use CSSTree Validator to "type check" CSS

### PR DESCRIPTION
**Pull request by forabi**
(see https://github.com/hollowverse/config/pull/83)

### CODE REVIEW

File: src/app/pages/NotablePerson/RelatedPeople.module.scss [hunk 1]
```diff
@@ -17,7 +17,6 @@
   -webkit-overflow-scrolling: touch;
 
   overflow-x: scroll;
-  overflow-x: overlay;
```
Comment (forabi) [inline src/app/pages/NotablePerson/RelatedPeople.module.scss:20]: This is a non-standard property, only supported in WebKit and it seems to have no effect in recent versions of Safari.

File: src/app/components/TextPage/TextPage.module.scss [hunk 2]
```diff
@@ -53,7 +53,6 @@
   a {
     color: $color-blue;
     text-decoration: none;
-    text-decoration-skip: ink;
```
Comment (forabi) [inline src/app/components/TextPage/TextPage.module.scss:56]: This is already set in global CSS.

File: src/app/components/NavBar/NavBar.module.scss [hunk 3]
```diff
@@ -7,7 +7,6 @@
       background: linear-gradient(50deg, #4cfde9 0%, #fbc954 50%, #4435f7 120%);
       // stylelint-disable-next-line property-no-vendor-prefix
       -webkit-background-clip: text;
-      background-clip: text;
```
Comment (forabi) [inline src/app/components/NavBar/NavBar.module.scss:10]: Non-standard property, unprefixed version is not supported by any browser.

File: src/app/components/App/App.module.scss [hunk 4]
```diff
@@ -38,7 +38,7 @@
   }
 
   a {
-    text-decoration-skip: ink;
```
Comment (forabi) [inline src/app/components/App/App.module.scss:41]: The property name has been changed in the spec.

**Review by wholesomedev — COMMENTED** [review]
LGTM

**Pull request merged**